config/install/tide_webform.defaults.yml

```diff
@@ -0,0 +1,8 @@
+privacy_statement:
+  heading: Privacy statement
+  content: |
+    <p>The Department collects the information that you provide with this complaint form. The information that you provide is used to respond to your enquiry. You can request access to, and corrections of, any personal information provided in this form. Requests for access or correction should be sent to [site:mail].</p>
+    <p>The Department is using the services of Drupal 8 to administer this form. The information that you provide is stored in our Drupal 8 content management system and Amazon Web Services servers.</p>
+    <p>For more information on the Department’s handling of any personal information, please refer to the Department’s Privacy Statement.</p>
+    <p>If you do not provide all or part of the information requested, the Department may not be able to contact you regarding your enquiry. Please contact the Department if you wish to use an alternative mechanism to the online form.</p>
+  agreement: I have read and understand the privacy statement.
```

Comment:
Why not just call this `title` as that's what it maps to?

Reply:
because logically it's not the title. Webform element renders it as the title of the checkbox, and heading/content as description of the checkbox.